crates/storage/src/indexed.rs

```diff
@@ -70,7 +65,7 @@ where
     where
         T: 'b,
     {
-        self.no_prefix().range_raw(store, min, max, order)
+        self.primary.range_raw(store, min, max, order)
```

Comment:
Instead of creating a `self.no_prefix` method and working with the `Prefix`, we now simply delegate to `self.primary`.